packages/inspection-report/src/hooks/useCurrency.js

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useCallback, useContext, useMemo, useState } from 'react';
+
+/**
+ * Currency context which will create wrapper for currency manipulation.
+*/
+export const Context = createContext({
+  currency: '$',
+  updateCurrency: () => { }
+});
+
+/**
+ * Currency wrapper used to abstract currency functionality.
+ *
+ * @return {React.ReactNode}
+*/
+export function CurrencyProvider({ children }) {
+  const [currency, setCurrency] = useState('$');
+
+  /**
+   * Set current currency.
+   *
+   * @param value Value of currency
+   * @return {void}
+  */
+  const updateCurrency = useCallback((value) => setCurrency(value), []);
```

Comment:
No need to to create this function and use `useCallback` here. You can simple add `updateCurrency: setCurrency` in context value.

Reply:
Done